ui/src/app/pages/analysis/leonardo-app-launcher.tsx

```diff
@@ -460,17 +460,6 @@ export const LeonardoAppLauncher = fp.flow(
       return appendNotebookFileSuffix(this.getNotebookName());
     }
 
-    private async initializeNotebookCookies() {
-      return await this.runtimeRetry(() =>
```

Comment:
Why were we retrying this, and should I keep that functionality? An error modal telling the user to refresh on the first failure may be sufficient.